ui/src/app/views/workspace-list/component.ts

```diff
@@ -101,4 +111,15 @@ export class WorkspaceListComponent implements OnInit, OnDestroy {
   addWorkspace(): void {
     this.router.navigate(['workspace/build'], {relativeTo : this.route});
   }
+
+  get twoFactorBannerEnabled() {
+    if (this.firstSignIn == null) {
+      return false;
+    }
+    // If it is three days or more after they first sign in, deactivate the banner
+    if (new Date().getTime() - this.firstSignIn.getTime() > 2.592e+8) {
```

Comment:
nit: Would prefer something more understandable for this constant, like 3 * 24 * 60 * 60 * 1000

Reply:
That is way better! Will change. (And I actually checked our current domain, and there's a seven day grace period, so I'll move to that.)